docs/user/FAQ.md

```diff
@@ -1,5 +1,69 @@
 # FAQ
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications. 
```

Comment:
Maybe add: The Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS standard to describe the deployment and management of applications in a portable manner. Based on standard-compliant TOSCA runtimes, such as the OpenTOSCA ecosystem, the deployment and management can be automated.